tests/ethernet/eth.py

```diff
@@ -0,0 +1,23 @@
+import utils
+
+
+def captures_ok(api, cfg, size, src, dst):
```

Comment:
after moving it to utils, see if we can call it like so:
```python
utils.eth.captures_ok()
```

It's not impossible, but if it's too much effort for now, consider renaming it to add prefix.